src/client/components/general/Ingredients/index.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import './ingredients.scss'
+import {ingredients} from '../../../utils/const'
+import IngredientCard from '../IngredientCard'
+
+function index() {
```

Comment:
Change the index functions names in every component of the application to the name of the parent folder